modules/mirror-registry-remote.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-remote_{context}"]
+= Mirroring on a remote host with mirror registry for {product-title}
+
+Remote mirroring with mirror registry for {product-title}
+
+This procedure introduces how to install Red Hat Quay on a remote host using the `mirror-registry` tool. By doing so, users can create a registry to hold a mirror of {product-title} images.
+
+[NOTE]
+====
+Installing Red Hat Quay using the `mirror-registry` CLI tool makes several changes to your machine. After installation, a `/etc/quay-install` directory is created, which has installation files, local storage, and the configuration bundle. Trusted SSH keys are generated in case the deployment target is the local host, and systemd files on the host machine are set up to ensure that container runtimes are persistent. Additionally, an initial user named `init` is created with an auto-generated password. All access credentials are printed at the end of the install routine.
+====
+
+.Procedure
+
+. Download the `mirror-registry.tar.gz` package for the latest version of mirror registry found on the link:https://console.redhat.com/openshift/downloads[*Downloads*] page of {product-title}.
+
+. Install Red Hat Quay on a remote host using the `mirror-registry` tool:
++
+[source,terminal]
+----
+$ sudo ./mirror-registry install -v \
+  --targetHostname <host_example_com> \
+  --targetUsername <example_user> \
+  -k ~/.ssh/my_ssh_key \
+  --quayHostname <host_example_com> \
+  --quayRoot <example_directory_name>
+----
+
+. The Red Hat Quay console is accessible at `\https://<host.example.com>:8443` after installation. Use the user name and password generated during installation to log into the registry by running the following command:
```

Comment:
```suggestion
. The mirror registry console is accessible at `\https://<host.example.com>:8443` after installation. Use the user name and password generated during installation to log into the registry by running the following command:
```

Reply:
Changed to match the local install login procedure.